Formula/qt.rb

```diff
@@ -166,6 +167,9 @@ def install
       -no-sql-mysql
       -no-sql-odbc
       -no-sql-psql
+
+      -I#{Formula["molten-vk"].opt_include}
+      -I#{Formula["molten-vk"].opt_libexec}/include
```

Comment:
It needs to detect dependencies on it's own, this is not a proper solution

Reply:
It's the only way it works, and it the recommended way according to this: https://www.qt.io/blog/2018/05/30/vulkan-for-qt-on-macos

Reply:
Should work without if `find_package(Vulkan)` works. I'm guessing that doesn't?